pkg/cmd/develop/develop.go

```diff
@@ -125,11 +125,17 @@ func parseArguments(cmd *cobra.Command) []string {
 		}
 	}
 
-	tpCmd := append([]string{
+	tpArgs := []string{
 		"--deployment", cmd.Flag("deployment").Value.String(),
-		"--expose", cmd.Flag("port").Value.String(),
 		"--method", cmd.Flag("method").Value.String(),
-		"--run"}, runArgs...)
+	}
+	if s, err := cmd.Flags().GetString("port"); err == nil && s != "" {
```

Comment:
I think that should work too:

```suggestion
	if cmd.Flags().Changed("port") {
```